test/check-machines-networks

```diff
@@ -868,6 +868,9 @@ class TestMachinesNetworks(VirtualMachinesCase):
 
             def cleanup(self):
                 b.click(f"#delete-network-{self.network_name}-{self.connection_name}-ipv4-dhcp-host-{self.entry_id}-button")
+                b.wait_in_text(".pf-c-modal-box__body .pf-c-description-list", self.ip)
+                b.wait_in_text("#delete-resource-modal-IP", self.ip)
+                b.wait_in_text("#delete-resource-modal-MAC", self.mac)
```

Comment:
This would better be squashed into the "Redesign vNIC removal dialog" commit.

Reply:
This is a completely different dialog. Removing vNIC and DHCP static entries from virtual networks are different functionalities with different dialogs.